almalinux-deploy.sh

```diff
@@ -546,6 +546,19 @@ restore_issue() {
     save_status_of_stage "restore_issue"
 }
 
+# Cleanup sssd(8) cache
+cleanup_sss_cache() {
+    if get_status_of_stage "cleanup_sss_cache"; then
+        return 0
+    fi
+    if [[ "${DOWNGRADE}" == 'YES' ]]; then
+        for file in /var/lib/sss/db/cache_*.ldb; do
```

Comment:
It fails if no cache files exist:
`mv: cannot stat '/var/lib/sss/db/cache_*.ldb': No such file or directory`

I propose to loop like:
`for file in $( find /var/lib/sss/db/ -name cache_\*.ldb -print ); do`